apps/prairielearn/elements/pl-file-preview/pl-file-preview.mustache

```diff
@@ -5,9 +5,9 @@
   <div class="card mb-2">
     <div class="card-header">Files</div>
     {{#has_files}}
-    <ul class="list-group list-group-flush">
+    <div class="list-group list-group-flush">
```

Comment:
I changed this so that any lists in a notebook preview would render at the correct depth.